code/planning/src/local_planner/motion_planning.py

```
@@ -369,7 +418,7 @@ def map_corner(dist):
             elif dist < 50:
                 return 7
             else:
-                8
+                8  # TODO add return
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)